content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -8,9 +8,9 @@ description: Configure and schedule GPUs for use as a resource by nodes in a clu
 
 <!-- overview -->
 
-{{< feature-state state="beta" for_k8s_version="v1.10" >}}
+{{< feature-state state="stable" for_k8s_version="v1.26" >}}
 
-Kubernetes includes **experimental** support for managing GPUs
+Kubernetes includes **stable** support for managing GPUs
 (graphical processing units) across several nodes.
```

Comment:
Optional extra tweak:
```suggestion
Kubernetes supports managing GPUs (graphical processing units) across
different nodes in your cluster, using
{{< glossary_tooltip text="device plugins" term_id="device-plugin" >}}.
```
(and then remove the later use of that glossary tooltip shortcode).